### PR TITLE
feat: update cli and rename to views-local

### PR DESCRIPTION
**Pull request by dariocravero**
@neesonch @eugenserbanescu check this out, it adds the CLI part of it.

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -1,3 +1,122 @@
...
+  Object.keys(i18n).forEach(lang => {
+    fs.writeFileSync(
+      path.join(translationsRoot, `${lang}.json`),
+      JSON.stringify(i18n[lang], null, ' '),
+      'utf-8'
+    )
+  })
+}
+
+async function set({ defaultLanguage, translationsRoot, viewsRoot }) {
```
Comment (eugenserbanescu) [inline cli.js:44]: No defaulting on the `defaultLanguage`?

File: cli.js [hunk 2]
```diff
@@ -1,3 +1,122 @@
...
 
-require('./index.js')(process.argv[2] || 'src')
+const glob = require('fast-glob')
+const fs = require('fs')
+const fse = require('enfsensure')
+const path = require('path')
+const getI18n = require('./get-i18n.js')
+const setI18n = require('./set-i18n.js')
+
+async function get({ defaultLanguage, translationsRoot, viewsRoot }) {
```
Comment (eugenserbanescu) [inline cli.js:10]: same as comment below on `defaultLanguage`

File: cli.js (hunk 1 above)
Comment (dariocravero) [reply cli.js:44]: we're defining it together with the other params here https://github.com/viewstools/local/pull/3/files#diff-2cce40143051e25f811b56c79d619bf5R77

Comment (eugenserbanescu) [reply cli.js:44]: Right you are! Sorry, didn't notice that :)

Comment (dariocravero) [reply cli.js:44]: no worries :)

**Review by eugenserbanescu — APPROVED** [review]

**Pull request merged**